examples/graph/app.js

```diff
@@ -162,34 +163,51 @@ class Root extends Component {
     }
   }
 
-  _onDragMove(el) {
+  _onMouseDown(event) {
+    // Use DeckGL.queryObject() to find the object under the mouse,
+    // and store it for updating on mouse move.
+    const point = {x: event.clientX, y: event.clientY};
+    const el = this.deckGL.queryObject(point);
     if (el) {
-      const {viewport} = this.state;
-      const {width, height} = viewport;
-      const x = el.x - width / 2;
-      const y = el.y - height / 2;
       this.setState({
         clicked: el,
         dragging: {
```

Comment:
Do you still need this nesting `dragging:{ node: }`
Maybe just `dragNode` and `dragPoint`?